frontend/src/components/DataCard/ConfirmationModal.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { Button, Modal } from "react-bootstrap";
+
+const ConfirmationModal = ({ fn, showModal, setSelectedLabelID }) => {
```

Comment:
The only issue I'm seeing here is that if we are in the history table and check the "unlabeled data" check (only will show up for projects without IRR turned on, so not the default projects) the confirmation popup still shows when you're labeling that data even though the data does not have a prior label.